PixelHunter1995/Content/SmurphFiles/dialog_intro.smph

```diff
@@ -0,0 +1,95 @@
+#IntroInit
+Nick: OK, so I've started the boiler and poured up the cooking oil.
+      What's the next ingredient?
+Ursula: Let's see here...
+        Next, the recipe tells you to add charcoal, iron and salt.
+Nick: Charcoal, iron and salt, check! Felixia, could you fetch those for me?
+Felixia: Um, OK...
+<goto IntroNickOptions>
+
+#IntroNick
+Ursula: Hey Nick, how's it bubbling?
+Nick: Could you get the remaining ingredients, please?
+<goto IntroNickOptions>
+
+#IntroNickOptions
+- [!found_salt] Where can I find salt?
+  Felixia: Where can I find salt?
+  Nick: There should probably be something around here containing salt.
+- [!found_charcoal] Where can I find charcoal?
+  Felixia: Where can I find charcoal?
+  Nick: No idea.
+- [!found_iron] Where can I find iron?
+  Felixia: Where can I find iron?
+  Nick: We have a box of forks from the cafeteria somewhere around here.
+- Why should I do it? Go fetch it yourself!
+  Felixia: Why should I do it? Go fetch it yourself!
+  Nick: Hey, someone needs to overlook this boiler!
+        We don't want an accident in the club room, if we want to keep it...
+  <removeOption>
+- What are we even making here?
+  Felixia: What are we even making here?
+  Ursula: The elixir of life, of course!
+  Felixia: Oh, of course...
+  <removeOption>
+- Let me look around.
+  <exit>
+
+#IntroNickChips
+Felixia: Hey, do you think Alex's chips contain enough salt?
+Nick: I guess we can try?
+<removeFromInventory chips>
+
+#IntroNickFork
+Felixia: Here's a stainless steel fork.
+Nick: Nice, that's one of the forks I stole from the cafeteria!
+      It should contain some iron.
+<removeFromInventory fork>
+
+#IntroNickCharcoal
+Felixia: Here are Ursula's drawing pencils.
+         They should contain enough charcoal.
+<removeFromInventory charcoalPencils>
+
+#IntroNickConk
+Felixia: I got Alex to give up his can of Conk.
+Nick: Cool.
+<removeFromInventory conk>
+// TODO: Trigger animations and dialog ending the first puzzle.
+
+#IntroUrsula
+Felixia: Hey, Ursula.
+Ursula: Hi, Felixia.
+- [!found_salt] Where can I find salt?
+  Felixia: Where can I find salt?
+  Ursula: If you want the salt enough, the salt will find you.
+  Felixia: Um, OK...
+- [!found_charcoal] Where can I find charcoal?
+  Felixia: Where can I find charcoal?
+  Ursula: Don't you have any charcoal yourself?
+  - Not really?
+    Felixia: Not really?
+    Ursula: Well, OK...
+            I guess you can take my drawing pencils.
+    <addToInventory charcoalPencils>
+    <removeBaseOption>
```

Comment:
How do we define which option level we want to remove? Do we need to name the option somehow?

Reply:
Will it not remove itself since we have charcoal now?

Reply:
That's kind of true, at least if the `found_charcoal` variable is set.